exposed-kotlin-datetime/src/test/kotlin/org/jetbrains/exposed/sql/kotlin/datetime/DefaultsTest.kt

```diff
@@ -576,4 +576,30 @@ class DefaultsTest : DatabaseTestsBase() {
             }
         }
     }
+
+    @Test
+    fun testColumnOnUpdateCurrentTimestamp() {
+        val tester = object : Table("tester") {
+            val amount = integer("amount")
+            val created = timestamp("created").defaultExpression(CurrentTimestamp).withDefinition("ON UPDATE", CurrentTimestamp)
+        }
+
+        withTables(excludeSettings = TestDB.ALL - TestDB.ALL_MYSQL_LIKE.toSet(), tester) {
+            assertTrue { SchemaUtils.statementsRequiredToActualizeScheme(tester).isEmpty() }
```

Comment:
Actually, I don't know how it's supposed to work with `statementsRequiredToActualizeScheme()`, should it return something if new `definition` was added, but the following test fails:

```
@Test
fun testUpdateExistingTable() {
    withDb {
        val before = object : IntIdTable("UpdateExistingTable") {
            val field = integer("field")
        }
        SchemaUtils.create(before)

        val after = object : IntIdTable("UpdateExistingTable") {
            val field = integer("field").withDefinition("ON UPDATE", CurrentTimestamp)
        }
        assertTrue(SchemaUtils.statementsRequiredToActualizeScheme(after).isNotEmpty())
    }
}
```

Reply:
In the ideal world, I think 2 things should happen with `statementsRequiredToActualizeScheme()`:

1. Calling multiple times on an unchanged column should generate no ALTER statements.
2. Calling on a column with added/removed `definition` should generate an ALTER statement.

But this becomes tricky without knowing exactly what the definition is and because not many special column definitions are returned by metadata (maybe default variants, visibility, comments in this example). Given this I'm not sure how it could be guaranteed using existing table column in database.

Migration script at least can be edited with correctly generated new sql from `after.field.modifyStatement()`.